Tiltfile

```diff
@@ -193,34 +195,68 @@ def launch_node_name_service(auto_init=True):
 def launch_node_year_service(auto_init=True):
   launch_node_svc("year-node", dirname="node/year-service", auto_init=auto_init)
 
-# Launch one of each of these types of services. Go services init by default
-# launch_go_frontend()
-# launch_python_frontend()
-# launch_ruby_frontend()
-# launch_java_frontend()
-# launch_dotnet_frontend()
+
+# Launch all services so that all service resources are registered with Tilt
+launch_go_frontend()

```

Comment:
Since we launch all services now, we may want to consider changing the print line that exists in each of the definitions. If you check out the Tiltfile output, it shows all the services starting which is not technically true anymore... 🤔  might need to ponder how best to achieve that.

```shell
About to start frontend-go with command cd golang/frontend && go run main.go -debug 
About to start frontend-rb with command cd ruby/frontend && bundle exec rackup ./frontend.ru
About to start frontend-java with command cd java/frontend && gradle bootRun
...
```

Reply:
Ohhhh yes this is a really good point, let me think on it and figure out a good solution. How important is it that we print the specific command it's starting up with?

Reply:
@JamieDanielson I ended up putting conditionals into the `launch_<lang>_svc` functions to only print out if those services are actually about to start. I also added a `launch_python_svc` function since it didn't exist before and made sure to test that all the python services work as expected.